src/design-system/components/Lens/Lens.tsx

```diff
@@ -35,7 +37,7 @@ export function Lens({
         e.key === shortcuts.global.SELECT.key ||
         (e.key === shortcuts.global.FORWARD.key && forwardNav)
       ) {
-        simulateClick(containerRef?.current, { bubbles: false });
+        simulateClick(containerRef?.current, { bubbles: bubbles });
```

Comment:
```suggestion
        simulateClick(containerRef?.current, { bubbles });
```

Reply:
with the new descriptive prop name I don't think i can do this